using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +148,60 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
 currently support the `FILTER` clause even though it is possible to
 parallelize but we might add support for this in a future version.
 
+By default, a continuous aggregate is refreshed when created, which
+means that it is immediately populated with data. Since this can be a
+lengthy process it is sometimes desirable to defer materialization to
+a manual refresh or a policy that does it in the background. You can
+disable the immediate materialization of data using the `WITH NO DATA`
+option to `CREATE MATERIALIZED VIEW`.
+
+#### Automatic refreshing with a continuous aggregate policy
+
+You can refresh the continuous aggregate view manually as above, but
+it is also possible to automate the refreshing by adding a continuous aggregate 
+policy with [`add_continuous_aggregate_policy`][add-continuous-aggregate-policy]. 
+For example:
```

Comment:
Could you elaborate what the policy does? Something along the lines of:  This policy would refresh the continuous aggregate from the beginning until upto 2 hours ago.

Reply:
I added a comment before, but it is already outlined after the query, so it does look redundant.